tests/e2e/e2e_workflow.spec.js

```diff
@@ -192,6 +192,120 @@ describe('Basic end-to-end Workflow', function () {
       });
   });
 
+  it('Installs sass support successfully via `ng install sass`', function() {
+    this.timeout(420000);
+
+    sh.exec('npm install node-sass', { silent: true });
```

Comment:
@jkuri, I didn't know about `ng install` :smiley:
I try to follow along, but you are moving too fast :+1: